common/src/main/scala/explore/model/enums/SEDType.scala

```diff
@@ -47,29 +47,28 @@ sealed abstract class SEDTypeEnum[T](
   import SpectralDefinition.*
 
   private def toBandNormalized[T](
-    sed: UnnormalizedSED
+    sed: Option[UnnormalizedSED]
```

Comment:
I think this change is not necessary after all. It's always called with `.some` and it's more compact to have it here than in all the callers.

The `None` case is handled somewhere else.